src/reason_pprint_ast.ml

```diff
@@ -630,9 +630,6 @@ let rules = [
     (TokenPrecedence, (fun s -> (Right, s = "asr")));
   ];
   [
-    (TokenPrecedence, (fun s -> (Left, s.[0] == '#')));
```

Comment:
Its precedence is below bang though https://github.com/facebook/reason/blob/499c11d886913c5a499d8e5f6d2a606386675440/src/reason_parser.mly#L1051, which is why I put it in 2nd place. (This diff is old too).